### ISSUE TITLE
allow command line execution 

### ISSUE THREAD

**Issue by kgk** [ahupp/python-magic#337]
I tried `python -m magic a_test_file`  but it failed.

**Comment by ddelange** [comment]
[python-magic](https://github.com/ahupp/python-magic) is a Python interface to the libmagic file type identification library. libmagic identifies file types by checking their headers according to a predefined list of file types. This functionality is exposed to the command line by the Unix command [`file`](https://www.darwinsys.com/file/).

[file tldr page](https://tldr.inbrowser.app/pages/common/file)
[file man page](https://manned.org/file)
[file pkgs page](https://pkgs.org/download/file)

**Comment by kgk** [comment]
absolutely, but sometimes this is installed and file is not.. just though 
`python -m magic somefile` would be usefull in this case
